fontforgeexe/fontinfo.c

```diff
@@ -8433,16 +8447,6 @@ return;
     vgcd[13].gd.u.list = ibmfamily;
     vgcd[13].creator = GListButtonCreate;
 
```

Comment:
Tabbing does not work for me, either, as it happens, and you are right that the order is already broken anyway. So it's probably not worth fixing in this pull request.